packages/backend/src/services/RenameService/rename.test.ts

```diff
@@ -0,0 +1,923 @@
+import {
```

Comment:
tests are important on this feature, not only to test methods and factories, but also it should help with scalability. 

if we introduce a new field to eg:  cartesian chart, this should complain, and force us to fix it.